doc/whats_new.rst

```diff
@@ -16,6 +16,10 @@ NEW
   The resulting average model is the one used as a classifier or a regressor.
   These two objects also leverage the `NiftiMaskers` to provide a direct
   interface with the Nifti files on disk.
+- New plotting function
+  `nilearn.plotting.plot_carpet` generates a "carpet plot" (also known
```

Comment:
This should have been :func:`nilearn.plotting.plot_carpet` to have a link to the API doc. I still that I'll merge and change this in master, to save everybody some time.